generator/generator.go

```diff
@@ -869,3 +878,76 @@ func getNamespace(ns string) string {
 
 	return "default"
 }
+
+// Set default psp of generator. Default PSP file has priority over setDefaultPsp().
+func (pg *Generator) SetDefaultPspFromFile(defaultPspFileName string) error {
+	var psp policyv1beta1.PodSecurityPolicy
+
+	pspObjFile, err := os.Open(defaultPspFileName)
+	if err != nil {
+		return fmt.Errorf("Could not open psp object file %s for reading: %v", defaultPspFileName, err)
+	}
+	defer pspObjFile.Close()
+
+	pspObjString, err := ioutil.ReadAll(pspObjFile)
+
+	if err != nil {
+		return fmt.Errorf("failed to read contents of psp object file %s: %v", defaultPspFileName, err)
+	}
+
+	pspObjJson, err := yaml.YAMLToJSON([]byte(pspObjString))
+	if err != nil {
+		return fmt.Errorf("Could not parse psp Object: %v", err)
+	}
+
+	var anyJson map[string]interface{}
+
+	err = json.Unmarshal(pspObjJson, &anyJson)
+
+	if err != nil {
+		return fmt.Errorf("Could not unmarshal json document: %v", err)
+	}
+
+	decoder := json.NewDecoder(bytes.NewReader(pspObjJson))
+	decoder.DisallowUnknownFields()
+
+	if anyJson["kind"] != "PodSecurityPolicy" {
+		return fmt.Errorf("Kind is not PodSecurityPolicy: %s", defaultPspFileName)
+	}
+
+	if err = decoder.Decode(&psp); err != nil {
+		return fmt.Errorf("Could not unmarshal json document as PSP: %v", err)
+	}
+
+	pg.defaultPsp = psp
+	// Set null fields.
+	pg.setDefaultPsp()
+
+	log.Debugf("setDefaultPspFromFile() is done. Default psp is : %v", &pg.defaultPsp)
+
+	return nil
+}
+
+// Set default PSP.
+func (pg *Generator) setDefaultPsp() error {
+	if pg.defaultPsp.Spec.RunAsUser.Rule == "" {
```

Comment:
This is duplicated with line 554

Reply:
Thanks for advise :)
I will remove duplication and make test. Please wait for a while time.

Reply:
I'm sorry but I didn't understand what I should do...  

Around line 932, set a default psp which `Generator` struct has. 
Around line 554, set a psp the default psp.  

I think these are like but not same... Is there way that decrease lines?